python/README.md

```diff
@@ -215,23 +327,23 @@ async def main():
             {"to": zero_address, "from": evm_account.address, "value": amount_to_send}
         )
 
-        # Get max fee and priority fee
         max_priority_fee = w3.eth.max_priority_fee
         max_fee = w3.eth.gas_price + max_priority_fee
 
         tx_hash = await cdp.evm.send_transaction(
             address=evm_account.address,
-            transaction={
+            transaction=DynamicFeeTransaction.from_dict(
```

Comment:
I suppose this would be a breaking change, but maybe we can sneak it in under the assumption that nobody is using this yet